int32 id
string name
int32 parent_marker
int32 child_marker
float64 confidence

KinematicState center
